resources/templates/settings.txt

```diff
@@ -1,25 +1,15 @@
 checkSignedStateFromDisk,                      1
 csvFileName,                                   MainNetStats
-csvOutputFolder,                               data/stats
```

Comment:
question: are the changes to this file backwards compatible?  Just want to make sure we won't indirectly stop supporting older versions.  I think previously we had 0.42.5 working.  Although, I don't really see it referenced in any of our e2e tests anymore.

Reply:
HEDERA_PLATFORM_VERSION used by e2e tests already has been changed to 'v0.49.0
so it should work with v0.49
If someone has to run some solo with on old version hedera repo, then the local settings.txt can be
modified or customized